src/pointer-analysis/value_set_analysis.h

```diff
@@ -52,14 +47,50 @@ class value_set_analysis_templatet:
       dest);
   }
 
+  virtual void initialize(const goto_programt &goto_program) override
+  {
+    baset::initialize(goto_program);
+    forall_goto_program_instructions(i_it, goto_program)
+    {
+      static_cast<domaint &>(this->get_state(i_it)).value_set.location_number =
+        i_it->location_number;
+    }
+  }
+
+  void analyse_all_functions(const goto_modelt &goto_model)
```

Comment:
Nit pick: the code base uses American spelling, even when most current contributors are based on this side of the Atlantic.